backends/dpdk/dpdkArch.cpp

```diff
@@ -1798,6 +1802,7 @@ getExternInstanceFromProperty(const IR::P4Table* table,
     }
 
     auto expr = property->value->to<IR::ExpressionValue>()->expression;
+    externName = expr->to<IR::PathExpression>()->path->name.name;
```

Comment:
originalName?

Reply:
Spec file uses the internal name instead of originalName